Examples/Algorithms/TrackFitting/src/GsfFitterFunction.cpp

```diff
@@ -111,38 +111,25 @@ std::shared_ptr<TrackFittingAlgorithm::TrackFitterFunction>
 ActsExamples::makeGsfFitterFunction(
     std::shared_ptr<const Acts::TrackingGeometry> trackingGeometry,
     std::shared_ptr<const Acts::MagneticFieldProvider> magneticField,
-    std::string lowBetheHeitlerPath, std::string highBetheHeitlerPath,
+    Acts::Experimental::AtlasBetheHeitlerApprox<6, 5> betheHeitlerApprox,
```

Comment:
would it make sense to use the typedef here?
```suggestion
    BHApprox betheHeitlerApprox,
```